spec/mailers/candidate_mailer_offers_and_rejections_spec.rb

```diff
@@ -121,13 +121,22 @@ def setup_application
     end
 
     describe '.application_rejected_all_rejected' do
-      it_behaves_like(
-        'a mail with subject and content', :application_rejected_all_rejected,
-        I18n.t!('candidate_mailer.application_rejected.all_rejected.subject', provider_name: 'Falconholt Technical College'),
-        'heading' => 'Dear Tyrell',
-        'course name and code' => 'Forensic Science (E0FO)',
-        'providers rejection reason' => 'The application had little detail.'
-      )
+      if FeatureFlag.active?('apply_again')
```

Comment:
Is this correct? What will have activated/disabled the feature flag here?

Reply:
yea that absolutely should not be like that. It should test with and without the flag on. I've fixed it.